src/app/pages/ArticlePage/PagePromoSections/RelatedContentSection/RelatedContentItem/index.test.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ServiceContextProvider } from '#app/contexts/ServiceContext';
+import { ToggleContextProvider } from '#contexts/ToggleContext';
+import RelatedContentItem from '.';
+import { RelatedContentData } from '../fixture';
+
+// eslint-disable-next-line react/prop-types
+const RelatedContentItemComponent = ({ fixtureData, service = 'mundo' }) => (
```

Comment:
from the standards `Avoid using the type of entity in naming.`

the `Component` suffix is not necessary

Reply:
tbh even `Fixture` is fine